ts/nni_manager/training_service/kubernetes/kubeflow/kubeflowTrainingService.ts

```diff
@@ -310,7 +312,7 @@ class KubeflowTrainingService extends KubernetesTrainingService implements Kuber
         // Generate kubeflow job resource config object
         const kubeflowJobConfig: any = await this.generateKubeflowJobConfig(trialJobId, trialWorkingFolder, kubeflowJobName, workerPodResources,
                                                                       nonWorkerResources);
-
+        this.log.info(`${JSON.stringify(kubeflowJobConfig)}`)
```

Comment:
Don't use `JSON.stringify`. Logger will format it. `JSON.stringify` could crash NNI manager if the content contains non-JSON values.